variables.tf

```diff
@@ -51,6 +51,22 @@ variable "platform_role_override_name" {
   default     = ""
 }
 
+################################################################################
+# IAM user
+################################################################################
+
+variable "platform_user_force_destroy" {
+  description = "Enable force destroy of the user"
+  type        = bool
+  default     = true
+}
+
+variable "platform_feature_user_enabled" {
```

Comment:
Please keep this variable as the first variable in the `# IAM user` section for brevity